pallets/Polkadex_Ido/src/lib.rs

```diff
@@ -554,7 +558,9 @@ decl_module! {
             ensure!(amount <= funding_round.max_allocation && amount >= funding_round.min_allocation, Error::<T>::NotAValidAmount);
 
             let current_block_no = <frame_system::Pallet<T>>::block_number();
-            ensure!(current_block_no < funding_round.close_round_block && current_block_no >= funding_round.start_block, <Error<T>>::NotAllowed);
+            // Ensure investors can only show interest during the voting period of the round
+            // Investors who shows interested are randomly selected (whitelisted)  to participate in the round at round start block
+            ensure!(current_block_no < funding_round.start_block, <Error<T>>::NotAllowed);
```

Comment:
Just confirm, this should be allowed until round starts?

Reply:
Yes, So that we can random select participants